CHANGES

```diff
@@ -3,6 +3,53 @@ Flask-Security Changelog
 
 Here you can see the full list of changes between each Flask-Security release.
 
+Version 3.1.0
+-------------
+
+Released TBD
+
+- 8919828 i18n: added Japanese translation
```

Comment:
@avilaton thanks! Can you please match the style with the rest of change log?

Reply:
I'm on it, I have a question, why was `next`removed here https://github.com/mattupstate/flask-security/commit/79545a92a1ebb601386fb34820508058ad7cdff6